src/config/app.config.ts

```diff
@@ -31,22 +37,46 @@ export default registerAs('app', (): Record<string, any> => {
     throw new InvalidConfigException(`LOG_LEVEL must be one of ${validLogLevels} or not specified.`);
   }
 
-  return {
-    name: process.env.APP_NAME || 'tfs',
-    env: process.env.NODE_ENV || 'development',
+  const versionPrefix = 'v';
```

Comment:
Is there a centralised constant location, if yes can this please be refered from there.